queries.sql

```diff
@@ -0,0 +1,3 @@
+SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
```

Comment:
- [ ] The execution time for these two queries highlighted added is below **1000ms (1 second)**. According to the requirements, the execution time of each of these queries should be above **1000ms( (1 second)**. I would advise you to run these queries two or three more times to ensure there are many entries in the table before you run the query 👍 😎 :

```sql
INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

#### Hint: Please update your screenshots after implementing this. 👍

Reply:
Thank you, will do and revert.

Reply:
## Updated screenshots of executive time exceeding 1000 ms

![image](https://github.com/tsheporamantso/vet-clinic-audit/assets/110445380/ee769547-9c38-4510-98c7-a79c1f32d0df)